crates/rolldown/tests/esbuild/default/export_forms_common_js/artifacts.snap

```diff
@@ -13,7 +13,7 @@ input_file: crates/rolldown/tests/esbuild/default/export_forms_common_js
 //#region a.js
 var abc;
 var init_a = __esmMin(() => {
-	abc = undefined;
+	abc = void 0;
```

Comment:
This does emit small output. But is it still meaningful in the aspect of bundling with not enabling minifer? If not, we need a option to control this behavior.